lang/python/python3-dbus/Makefile

```diff
@@ -0,0 +1,45 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-dbus
+PKG_VERSION:=1.2.18
+PKG_BUILD_DIR:=$(BUILD_DIR)/dbus-python-$(PKG_VERSION)
+
+PYPI_NAME:=dbus-python
+PKG_HASH:=92bdd1e68b45596c833307a5ff4b217ee6929a1502f5341bae28fd120acf7260
+PKG_MAINTAINER:=НЕВСКИЙ БЛЯДИНА <neva_blyad@lovecry.pt>
+
+PKG_INSTALL:=1
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+# The file included below defines PYTHON3_VERSION
+include ../python3-version.mk
+
+define Package/python3-dbus
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Python bindings for libdbus
+  URL:=http://www.freedesktop.org/wiki/Software/DBusBindings/#python
+  DEPENDS:= \
+    +glib2 \
+    +python3 \
+    +libdbus
```

Comment:
I don't think this is correct. You should check by following the documentation: https://github.com/openwrt/packages/blob/master/lang/python/README.md#python-package-dependencies